windshaft/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:vivid
```

Comment:
I've read in a couple places that the ubuntu base images are heavier than they need to be and that using debian base images is generally preferable, but I don't know if we've come to a company consensus on this. @hectcastro ?